docs/modules/ROOT/pages/jlama.adoc

```diff
@@ -0,0 +1,73 @@
+= Jlama
+
+include::./includes/attributes.adoc[]
+
+https://github.com/tjake/Jlama[Jlama] provides a way to run large language models (LLMs) locally and in pure Java, eventually also embedded in your Quarkus application.
```

Comment:
What "eventually"?

Reply:
You're right at the moment is the only why to make it work, I rephrased accordingly. I wonder if we could also talk via REST to an external server running Jlama and if this use case would actually make sense.

Reply:
We haven't tried it, but assuming that it implements the OpenAI protocol correctly, it should just work